.github/workflows/deep-test.yml

```diff
@@ -0,0 +1,105 @@
+name: deep test
+
+on: push
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  find-all-packages:
+    # Deep testing limited to personal fork
+    if: ${{ github.repository_owner == 'ViliusSutkus89' }}
```

Comment:
I don't think we have to do this otherwise you will always have to keep your repo up to date for testing if somebody else changes something.